src/screens/DiscoveryScreenV2/fragment/DomainFragment.js

```diff
@@ -278,18 +298,7 @@ const DomainFragment = ({
       </View>
     );
 
-  return (
-    <View>
-      {!withoutRecent && (
-        <RecentSearch
-          shown={isFirstTimeOpen}
-          setSearchText={setSearchText}
-          setIsFirstTimeOpen={setIsFirstTimeOpen}
-        />
-      )}
-      {__renderDomainItems()}
-    </View>
-  );
+  return <View>{renderDomainItems()}</View>;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The refactoring of the return statement to only


<!-- This is an auto-generated comment by OSS CodeRabbit -->